### ISSUE TITLE
RNA definition

### ISSUE THREAD

**Issue by AlasdairGray** [BioSchemas/specifications#282]
The new [RNA type](http://bio.sdo-bioschemas-227516.appspot.com/RNA) needs a descriptive description.

**Comment by folago** [comment]
Agreed at F2F meeting: Reference to a specific RNA molecule (Ribonucleic Acid). Examples: “pBABE-Puro, Retroviral Vector”, “Rosellinia necatrix fusarivirus 1 (RnFV1)”
